game/scripts/vscripts/libraries/timers.lua

```diff
@@ -84,11 +89,10 @@ end
 
 function Timers:start()
 	Timers = self
-	self.timers = {}
+	self.timers = {realTime = BinaryHeap("endTime"), gameTime = BinaryHeap("endTime")}
 	self.nextTickCallbacks = {}
-
-	--local ent = Entities:CreateByClassname("info_target") -- Entities:FindByClassname(nil, 'CWorld')
-	local ent = SpawnEntityFromTableSynchronous("info_target", {targetname="timers_lua_thinker"})
+	
+	local ent = Entities:CreateByClassname("info_target") -- Entities:FindByClassname(nil, 'CWorld')
```

Comment:
Is there a reason for this change? IIRC the only difference between these two is that `targetname` may be printed in console when some timer takes too much time, though it's not very helpful anyway.